--- chrome.manifest.orig	2014-03-12 11:31:41.960054153 -0300
+++ chrome.manifest.modif	2015-08-14 12:41:11.688896924 -0300
@@ -1 +1,13 @@
 manifest components/components.manifest
+
+# Desativa o about:config
+override chrome://global/content/config.xul data:text/html,Desativado
+
+# Desativa o about:addons
+override chrome://mozapps/content/extensions/extensions.xul data:text/html,Desativado
+
+# Desativa o about:preferences
+override chrome://browser/content/preferences/in-content/preferences.xul data:text/html,Desativado
+
+# Desativa o about:app-manager
+override chrome://browser/content/devtools/app-manager/index.xul data:text/html,Desativado
