--- plugin.c.orig	2013-05-13 14:26:32.538510796 -0300
+++ plugin.c.modif	2013-05-13 14:26:32.550510834 -0300
@@ -357,12 +357,6 @@
  * External so can be used from a plugin. */
 gboolean plugin_button_press_event(GtkWidget *widget, GdkEventButton *event, Plugin *plugin)
 {
-    if (event->button == 3)	 /* right button */
-    {
-        GtkMenu* popup = (GtkMenu*) lxpanel_get_panel_menu(plugin->panel, plugin, FALSE);
-        gtk_menu_popup(popup, NULL, NULL, NULL, NULL, event->button, event->time);
-        return TRUE;
-    }    
     return FALSE;
 }
 
